ndmg/utils/qa_utils.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+
+# Copyright 2016 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# qa_utils.py
+# Created by Wilson Tang
+
+import numpy as np
+
+def get_min_max(data, minthr=2, maxthr=95):
+    """
+    Parameters
+    -----------
+    data: np array
+        3-d regmri data to threshold.
+    minthr: int
+    maxthr: int
+    
+    Returns
+    -----------
+    min_max: tuple
+        tuple of minimum and maximum values
+    """
+    min_val = np.percentile(data, minthr)
+    max_val = np.percentile(data, maxthr)
+    
+    min_max = (min_val.astype(float), max_val.astype(float))
+    return min_max
+
+def opaque_colorscale(basemap, reference, vmin=None, vmax=None, alpha=1):
```

Comment:
make this function a bit more [sparse](https://gist.github.com/evandrix/2030615#file-pep20_by_example-py-L266), for readability

Reply:
check commit 09e7135

Reply:
great job